source/miscellaneous.rst

```diff
@@ -58,14 +58,46 @@ Go to ``Bharat-sim Master\\src\\test\\scala\\com.bharatsim.engine\\graph\\neo4j`
 Using args in main method
 ==========================
 While using the ``run`` command on the sbt shell, one can pass in some ``string`` arguments. These arguements can be called by the main function. 
-for example:
+For example, one might want to change the name of the output file everytime they run the code. Instead of changing the output name manually each time, one can write a code like the one given below.
+
 .. code::
 
     var outputName = "dummyName"
 
-    def main(args: Array[String]): Unit = {
+      def main(args: Array[String]): Unit = {
         outputName = args(0)
-         
-The above block of code is allocating a variable called \verb|outputName|, and then using the 1st string passed through the  ``run`` command. For example, ``run "example"``. In this case, ``outputName`` becomes ``"example"``. The variable can be used by the user where ever they need it. 
+        var beforeCount = 0
+        val simulation = Simulation()
+
+        simulation.ingestData(implicit context => {
+          ingestCSVData("C:\\Users\\prath\\Documents\\BharatSim\\src\\main\\resources\\citizen.csv", csvDataExtractor)
+          logger.debug("Ingestion done")
+        })
+
+        simulation.defineSimulation(implicit context => {
+          addLockdown
+          create12HourSchedules()
+
+          registerAction(
+            StopSimulation,
+            (c: Context) => {
+              getInfectedCount(c) == 0
+            }
+          )
+
+          beforeCount = getInfectedCount(context)
+
+          registerAgent[Person]
+
+          val currentTime = new Date().getTime
+
+          SimulationListenerRegistry.register(
+            new CsvOutputGenerator("C:\\Users\\prath\\Documents\\output\\control"+outputName+".csv", new SEIROutputSpec(context))
+          )
+        })
+
+
+The above block of code is allocating a variable called ``outputName``, and then using the 1st string passed through the  ``run`` command, as the ``outputName``. 
 
+To implement this, one must go to the ``sbt shell`` and type ``run "arguement"``. If this were to be used on the code-block above, the output csv file will be named as ``contolarguement.csv``.
```

Comment:
Instead of the "arguement", I would suggest using a more realistic name (say, ``MY_SIR_MODEL``) or something like that. You don't need to say that the file will be renamed, just that the user's string is now accepted in the code, and can be used by the modeller in different locations.

Also, please include a line about how to do this with the `java -jar` option!